src/anomalib/cli/utils/help_formatter.py

```diff
@@ -6,87 +6,77 @@
 import argparse
 import re
 import sys
+from typing import TypeVar
 
+import docstring_parser
 from jsonargparse import DefaultHelpFormatter
 from rich.markdown import Markdown
 from rich.panel import Panel
 from rich_argparse import RichHelpFormatter
 
-from anomalib.engine import Engine
-
 REQUIRED_ARGUMENTS = {
+    "train": {"model", "model.help", "data", "data.help", "ckpt_path", "config"},
     "fit": {"model", "model.help", "data", "data.help", "ckpt_path", "config"},
     "validate": {"model", "model.help", "data", "data.help", "ckpt_path", "config"},
     "test": {"model", "model.help", "data", "data.help", "ckpt_path", "config"},
     "predict": {"model", "model.help", "data", "data.help", "ckpt_path", "config"},
 }
 
-DOCSTRING_USAGE = {
-    "fit": Engine.fit,
-    "validate": Engine.validate,
-    "test": Engine.test,
-    "predict": Engine.predict,
-}
+try:
+    from anomalib.engine import Engine
+
+    DOCSTRING_USAGE = {
+        "train": Engine.train,
+        "fit": Engine.fit,
+        "validate": Engine.validate,
+        "test": Engine.test,
+        "predict": Engine.predict,
```

Comment:
do we need this for `export` as well? @djdameln, based on your proposal yesterday, we will still have the `export` subcommand, but integrate the export util functions to the model like `to_openvino`, `to_torch`, right?